schema/snapcraft.yaml

```diff
@@ -57,16 +57,6 @@ properties:
     uniqueItems: true
     items:
       - type: string
-  license:
-    type: string
-    description: path to a license file.
-  license-agreement:
-    type: string
-    description: action to take on licensing.
-    enum:
-      - explicit
-  license-version:
-    description: license version (used to re-trigger an agreement action).
```

Comment:
I don't think we need a warning, this should be a flat out breakage as if not it would create the expectation that it works when it clearly doesn't